setup.cfg

```diff
@@ -27,7 +27,7 @@ install_requires =
     numpy>=1.20
     scipy>=1.4.1
     sympy>=1.5,<=1.9
-    qiskit>=0.28, <=0.34
+    qiskit==0.38
```

Comment:
1. Have you checked if it works with `0.37`?
2. Perhaps we should do `~= 0.38.0` ? Or does `==0.38` already works like this?

Reply:
To test it out I would have to have someone with M1 macs try it. @SebastianMorawiec said that it would work with .38 so I decided pinning it was the easiest thing to do. =

Reply:
It's possible they could have had this working as recently as qiskit .36 because that was the most recent update to `qiskit-aer` (the package that failed to install) after the previous version we were using (qiskit .34).